.github/workflows/test.yml

```diff
@@ -63,6 +63,11 @@ jobs:
           fi
         id: build
 
+      - name: Run Forge build for scripts
+        if: matrix.profile != 'solc-0.7.6'
+        run: forge build --contracts ./scripts/
+        id: build-scripts
```

Comment:
I think this is a plurality issue. Forge by default includes `/script`, `/src`, and `/test` when building (as observed by `forge init temp && cd temp && forge build && ls -alh out/`).